src/reynir_correct/pattern.py

```diff
@@ -1493,8 +1992,8 @@ def create_patterns(cls) -> None:
             # Catch "Ég hafði ekki lagt mikið að mörkum."
             cls.add_pattern(
                 (
-                    "mark",  # Trigger lemma for this pattern
-                    "VP > { VP >> { 'leggja' } PP > { P > 'að' NP > { 'mark' } } }",
+                    "(mörk|mark)",  # Trigger lemma for this pattern
```

Comment:
No, this doesn't work. `frozenset(("mörk", "mark"))` works, however. See the function `lemma_match()` below.